india_banking_connector/host_to_host/doctype/citi_h2h_connector/citi_sftp_client.py

```
@@ -0,0 +1,137 @@
+		except Exception:
+			frappe.log_error(
+				"Citi SFTP Connection Failed", frappe.get_traceback(with_context=True)
+			)
+			return False
+
+	def upload(self, local_path: str, remote_path: str) -> bool:
+		try:
+			if not self.sftp:
+				self.connect()
+			self.sftp.put(local_path, remote_path, confirm=False)
+			return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)